### ISSUE TITLE
Domains: Mention our PWPO policy in checkout page when users attempting to purchase domains without paid plans

### ISSUE THREAD

**Issue by hacchism** [Automattic/wp-calypso#45743]
Most of our users who purchase their domains on their free plan sites don't know about our PWPO policy and get confused after domain purchases.

That results in more support requests where HEs need to explain about the PWPO policy and ask them to upgrade to paid plans so they can set their domains as primary. 

To reduce the hassle on both users and HEs, it would be great if we can mention the PWPO policy in the **Upgrade and save
section** of checkout page or add a popup to nudge users to add a paid plan.

![Screen Shot on 2020-09-18 at 14:59:30](https://user-images.githubusercontent.com/32432463/93561672-990c2080-f9bf-11ea-93f8-a2f41eae9744.png)

**Comment by KokkieH** [comment]
This has come up today in p2MSmN-8dL-p2

It would be good if a notice can also be added to https://wordpress.com/domains/add/use-your-domain/{SITE_URL} for accounts that aren't PWPO exempt.

**Comment by obenland** [comment]
@hacchism @KokkieH Could you point me to other places where we mention our PWPO policy? I'm not quite sure what that would all entail.

@kellychoffman @pablohoneyhoney Could I ask for your help with how to best display that PWPO policy in the checkout?

**Comment by kellychoffman** [comment]
Happy to help once I see what the PWPO is / entails.

**Comment by KokkieH** [comment]
@obenland most detailed info is here: P6Ya1B-Nh-p2 

tl;dr, we allow people to buy a domain as a stand-alone upgrade on free sites, but in order to set that domain as the primary address on the site, we require customers to also purchase a paid plan. 

I don't know if it still comes up regularly, but around the time this issue was created, we'd often have people buy a domain, then only find out they need a plan to set the domain as the primary address after the fact, with the result that they have to pay for the domain, as well as the full cost of the plan. Had they bought the plan from the start, the domain would have been included for free (the domain credit included in plans can't be applied retroactively if you buy a plan after you've bought a domain).

cc'ing @Automattic/nomado to confirm this is still accurate.

**Comment by delputnam** [comment]
This is still accurate.  Perhaps we should also review this policy.
